pkg/storage/storage.go

```diff
@@ -483,39 +586,17 @@ func (s *storageModule) createSubvol(size uint64, name string, poolType pkg.Devi
 			continue
 		}
 
-		candidates = append(candidates, Candidate{
+		candidates = append(candidates, candidate{
 			Pool:      pool,
 			Available: usage.Size - (reserved + size), // available after new subvolume
```

Comment:
Why is Available is compute with the new reserved size substracted ? We want to know how much storage is available **before** we reserve the new sub-volume.